client/client.js

```diff
@@ -85,15 +97,57 @@ Client.prototype.dataStore = function (message) {
   }
 
   if (changed) {
-    this.lastModified = Date.now();
-    Core.Persistence.persistKey(this.key, this, Client.dataTTL);
+    this._persist();
   }
 
   return true;
 };
 
+Client.prototype.dataLoad = function (callback) {
+  var self = this;
+
+  // When we touch a client, refresh the TTL
+  Core.Persistence.expire(self.key, Client.dataTTLGet());
+
+  // Update persisted subscriptions/presences
+  Core.Persistence.readKey(self.key, function (clientOld) {
+    if (clientOld) {
+      self.subscriptions = clientOld.subscriptions;
+      self.presences = clientOld.presences;
+
+      // Remove obsolete presences
+      Client._presencesDelete(self.presences, clientOld.id);
```

Comment:
why do we need to do this? They will be removed by the presence code anyway right?